02_bubble/alanhillgemann/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, bubble([]int{3, 2, 1, 5}))
+}
+
+func bubble(arr []int) []int {
```

Comment:
You are mutating the input slice. Please make a copy.
Ideally write a test that fails when checking if input has not been update, update your code until your test passes.

Reply:
Fixed